src/Helpers/Arr.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pest\Expectations\Helpers;
+
+/**
+ * Class Arr.
+ *
+ * Credits: most of this class methods and implementations
+ * belongs to the Arr helper of laravel/framework project
+ * (https://github.com/laravel/framework)
+ */
+final class Arr
```

Comment:
Add `internal` annotation to this class as we don't to support this class as kind of public API for pest 👍